x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/eql/create_eql_alert_type.ts

```diff
@@ -20,6 +20,7 @@ export const createEqlAlertType = (
   return {
     id: EQL_RULE_TYPE_ID,
     name: 'Event Correlation Rule',
+    ruleTaskTimeout: experimentalFeatures.securityRulesCancelEnabled ? '5m' : '1d',
```

Comment:
```suggestion
    ruleTaskTimeout: experimentalFeatures.securityRulesCancelEnabled ? '5m' : undefined,
```

Reply:
Similar for all rule types - can we leave `ruleTaskTimeout` unset if the experimental feature is not enabled?

Reply:
The alerting framework will set it to the alerting default if it is undefined for rule type, which is currently `5m`.

Reply:
One thing worth noting: the timeout is also used by the re-try mechanism in Task Manager. This mechanism is in place to resume rules that were running on a Kibana that suddenly crashed. For example, having a timeout of `1d` could take up to a day to continue a rule if ever Kibana crashed when it was running.